client/src/components/AddPlayer.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+class AddPlayer extends Component {
```

Comment:
this class does not hold state as well and would benefit from a speed boost using pure components 

```javascript 
import React from 'react';

export default (props) => {

  const handleSubmit = (event) => {
    event.preventDefault();
    const playerName = event.target.children[0].value;
    props.addPlayer(playerName);
  }

  return(
    <div>
      <form onSubmit={handleSubmit} >
        <input type="text" name="playerName" />
        <input type="submit" value="Add Player" />
      </form>
    </div>
  )
}
```